internal/overlord/servstate/handlers.go

```diff
@@ -431,6 +428,11 @@ func (s *serviceData) startInternal() error {
 	return nil
 }
 
+// buildCmdArgs returns the service command as a stream of arguments
+func (s *serviceData) buildCmdArgs() ([]string, error) {
+	return s.config.GetCommand(s.manager.serviceArgs[s.config.Name])
```

Comment:
Seems like we might as well just do this one-liner inline rather than have a method that just calls another method?

Reply:
That works as well.